src/ethsign/ethsign.go

```diff
@@ -649,24 +649,16 @@ func main() {
       Name: "import",
       Usage: "import hexadecimal private key into keystore",
       Flags: []cli.Flag{
-        cli.StringFlag{
-          Name:  "keystore",
+        cli.StringSliceFlag{
+          Name:  "key-store",
```

Comment:
I think I would prefer to keep the naming of the flag here compatible with previous versions.

Reply:
Gotcha, was thinking you'd want to make it consistent with all other commands though?

Reply:
ah yes, good point, fine to leave this as is then.